pg_tests/README.md

```diff
@@ -0,0 +1,23 @@
+# PostgreSQL Tests
+
+### Prerequisites
```

Comment:
- We should add that you need to pip(3) install psycopg2
- Are there other prerequisites for the test that aren't currently mentioned here?
	- Do you need to have psql installed for psycopg2 to work?
	- What about the account credentials for the database? Does your account already have to be present in postgres on your system for the account credentials to work properly? If so, can we add a bit of detail about how to do that?